src/bootstrap/src/core/debuggers/lldb.rs

```diff
@@ -0,0 +1,29 @@
+use std::path::PathBuf;
+
+use crate::core::builder::Builder;
+use crate::utils::exec::command;
+
+pub(crate) struct Lldb {
+    pub(crate) lldb_version: String,
+    pub(crate) lldb_python_dir: String,
+}
+
+pub(crate) fn discover_lldb(builder: &Builder<'_>) -> Option<Lldb> {
+    let lldb_exe = builder.config.lldb.clone().unwrap_or_else(|| PathBuf::from("lldb"));
```

Comment:
Suggestion: can you add a FIXME to #148361 for this? Trying to discover an `lldb` from PATH is highly questionable, because then you can pick up some completely random `lldb` 🤦